src/lib/components/ExpertBlock.tsx

```diff
@@ -13,6 +13,7 @@ export const ExpertBlock: React.FC<IExpertProps> = (props) => {
   return (
     <>
       <img
+        data-testid='expertBlock'
```

Comment:
you don't have to touch the components with testing data.
try decorating the components with the necessary attributes inside the test inself